experimental/relativetime/Cargo.toml

```diff
@@ -23,11 +23,21 @@ include = [
 ]
 
 [dependencies]
+displaydoc = { version = "0.2.3", default-features = false }
+icu_decimal = { version = "1.0.0", path = "../../components/decimal" }
 icu_provider = { version = "1.0.0", path = "../../provider/core", features = ["macros"] }
+icu_plurals = { version = "1.0.0", path = "../../components/plurals" }
+icu_displaynames = { path = "../displaynames" }
```

Comment:
Nit: You shouldn't need `icu_displaynames`

Reply:
I forgot to remove it from when I was debugging that cyclic dependency issue in icu_relativetime. Removed.